app/modules/linshare.share/components/quickShareComponent.js

```diff
@@ -0,0 +1,11 @@
+angular
+  .module('linshare.components')
+  .component('quickShare', {
+    template: require('./quickShare.html'),
+    controller: 'quickShareController',
+    controllerAs: 'quickShareVm',
+    bindings: {
+      shareObject: '=',
+      onAfterShare: '='
```

Comment:
```suggestion
      onAfterShare: '&'
```

Reply:
Changing the binding like this make the function not work